chain/tlbc/tlbc-spec.json

```diff
@@ -9,16 +9,51 @@
                     "multi": {
                         "0": {
                             "list": [
-                                "0x6d440f55d116DB555A2cB0e71a55Ec0ADab29154",
-                                "0x4767C6dfAa9b4F6286d27b6bb267F02389C83F56",
-                                "0xb5847679dEB6De72628Cd27921B72A677377dE55",
-                                "0xc13651500a405e2ebaea080c0190Ccc98D1dD442",
-                                "0x492bF3745d6129db640cE230845340f4fE8Bf1D5",
-                                "0x2e3B2E899Bdd6510Fd7955DE171a9bC185BD9D47",
-                                "0x34171678EB4ca49AE9B253EEBf42eb8C289c2722",
-                                "0x206E3c8e9980C0aC47826D2417C3709Ee57eFAbC",
-                                "0x9543D0F5212eaE6A9bf2E4AEDA2bD06eD5267EFb",
-                                "0x20274acDA6AC1615fF56b6F87d97aBBf97D0CC13"
+                                "0x3F5948DE91A72e6885F18EEC428F465d65e5A686",
```

Comment:
I thought we agreed that we should sort it to add some randomization.

Reply:
Please check again